### ISSUE TITLE
As a consumer, i would like to view another tab with "accolades"

### ISSUE THREAD

**Issue by sminke** [autorivalry/ar-render#42]
Can we create an extra tab for "Accolades" that allows us to list 3rd party awards like "Car & Driver 10 Best", etc.  If we don't think that this needs to be a standalone extra tab, maybe we just start adding it to the top of the Content/Text Box?  Thoughts either way?

**Comment by jordanskole** [comment]
@sminke can you please make sure to update the descriptions on these?

**Comment by sminke** [comment]
@jordanskole this is a request by omni, but i can't seem to find a way to add the label "user request" on this item

**Comment by jordanskole** [comment]
@sminke Right. But I have no idea what the title means. So can you please update the description with enough detail to understand?

**Comment by sminke** [comment]
@jordanskole I've been trying to update it... still working on figuring this out :)

**Comment by sminke** [comment]
The following image is from the bottom of a current Omni comparison page.  They like the images of the awards and accolades, they also like the "call to action" links/buttons...

http://i.imgur.com/YCKhyEG.png

**Comment by jordanskole** [comment]
@sminke "they also..." is a good indicator of a separate issue :wink:
